graylog2-web-interface/src/pages/DataNodePage.tsx

```diff
@@ -57,21 +59,102 @@ const BooleanValue = ({ value }: { value: boolean }) => (
   <><BooleanIcon name={value ? 'check-circle' : 'times-circle'} value={value} /> {value ? 'yes' : 'no'}</>
 );
 
+const ActionsCol = styled(Col)`
```

Comment:
We already have most of this code in the DataNodeAction component; we should reuse this component. We could introduce a prop to help us decide which button style we want when used on the page.